CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# dbt_zendesk_source v0.9.1
+
+
+# 🚨 New Schedule Holiday Table 🚨
+- Adding the `schedule_holiday` source table so that downstream models that involve business minutes calculations will accurately take holiday time into account. ([#92](https://github.com/fivetran/dbt_zendesk/issues/92))
```

Comment:
i would add that this can be disabled by `using_schedules`

Reply:
Updated